api-gateway/src/main/resources/static/scripts/inventory-product-list/inventory-product-list.template.html

```diff
@@ -52,14 +52,27 @@ <h2>Inventory Products</h2>
     </thead>
     <tr ng-repeat="product in $ctrl.inventoryProductList | filter:$ctrl.query track by product.productId">
         <td><span>{{product.productName}}</span></td>
-        <td><span>{{product.productId}}</span></td>
+            <td><span>{{product.productId}}</span></td>
         <td><span>{{product.productQuantity}}</span></td>
         <td><span>{{product.productPrice}}</span></td>
         <td><span>{{product.productDescription}}</span></td>
         <td><span>{{product.productSalePrice}}</span></td>
+
+        <td>
+            <a href="#!/inventory/{{$ctrl.inventory.inventoryId}}/products/{{product.productId}}">
+                Get Product Details
+            </a>
+
+        </td>
+
+
         <td><a ui-sref="updateProductInventory({inventoryId: $ctrl.inventoryProductList[0].inventoryId, productId: product.productId})">
-            <button class="add-bundle-button btn btn-success">Edit</button>
+            <button class="add-bundle-button btn btn-warning">Edit</button>
```

Comment:
Can elaborate on why you changed the btn-success to the btn-warning?

Reply:
My pleasure, so I changed the color from btn -success to btn - warning, because green means success and you haven't changed or modified any value yet, thats why i changed the btn color to yellow to show that you need to test the functionality of the button.

Reply:
I see thank you for informing me. It does make sense now because it will also match better with the inventory since the other update button in the inventories has it on "Warning" as well now that I look at it.